numpyro/examples/datasets.py

```diff
@@ -27,6 +27,16 @@
 
 dset = namedtuple("dset", ["name", "urls"])
 
+BART = dset(
+    "bart",
+    [
+        "https://github.com/juanitorduz/datasets/blob/add_bart_parts/bart/bart_0.npz?raw=true",
+        "https://github.com/juanitorduz/datasets/blob/add_bart_parts/bart/bart_1.npz?raw=true",
+        "https://github.com/juanitorduz/datasets/blob/add_bart_parts/bart/bart_2.npz?raw=true",
+        "https://github.com/juanitorduz/datasets/blob/add_bart_parts/bart/bart_3.npz?raw=true",
```

Comment:
This works locally :)

Once we merge https://github.com/pyro-ppl/datasets/pull/1 I can update the path to the `pyro-ppl` repo.

Reply:
changed the paths after https://github.com/pyro-ppl/datasets/pull/1